usr/share/lib/img_proof/tests/conftest.py

```diff
@@ -146,31 +146,27 @@ def f():
 
 
 @pytest.fixture()
-def is_suma_server(host, get_baseproduct):
+def is_suma_server(host):
     def f():
         suma_server_product = '/etc/products.d/SUSE-Manager-Server.prod'
         suma_server = host.file(suma_server_product)
-        base_product = get_baseproduct()
 
         return all([
             suma_server.exists,
-            suma_server.is_file,
-            base_product == suma_server_product
```

Comment:
This should be version dependent since we still need to support SUMa 4.3 for some time. For SUMa 4.3 the baseproduct link _must_ point to the SUMa product file.

Reply:
Agree, we could pass in a version to this fixture and add a conditional on that.